<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="TableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>501</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>TID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CPU Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priority</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="cteateThread">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Create </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb16">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>16</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rwlock">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>rwlock</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="condvar">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>conditional variables</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startAll">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>start all</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
